subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/ui/SubscriptionsWebViewActivity.kt

```diff
@@ -144,9 +149,24 @@ class SubscriptionsWebViewActivity : DuckDuckGoActivity() {
             is SubscriptionSelected -> selectSubscription(command.id)
             is ActivateOnAnotherDevice -> activateOnAnotherDevice()
             is RestoreSubscription -> restoreSubscription()
+            is GoToITR -> goToITR()
+            is GoToPIR -> goToPIR()
+            is GoToNetP -> goToNetP()
         }
     }
 
+    private fun goToITR() {
+        Toast.makeText(this, "Go To ITR", Toast.LENGTH_SHORT).show()
+    }
+
+    private fun goToPIR() {
+        Toast.makeText(this, "Go To PIR", Toast.LENGTH_SHORT).show()
+    }
+
+    private fun goToNetP() {
+        globalActivityStarter.start(this, NetPWaitlistInvitedScreenNoParams)
```

Comment:
@marcosholgado you should use the Activity params passed in the command here than doing this.

Reply:
I knew I did that for a reason :D, changed.